infra/helm-charts/mongo/charts/ingestbackup/compose-backup.yml

```diff
@@ -15,6 +15,6 @@ services:
     - BACKUP_DIR=dev
     - MONGO_HOST=mongo
   mongo:
-    image: mongo:3.4.3
+    image: mongo:4.2
```

Comment:
Are we intentionally downgrading the mongo version?

Reply:
it's an upgrade :)

Reply:
ah, sorry, I can't read. Probably need to change or clean my glasses 😂